modules/xmpp/xmpp.js

```diff
@@ -261,7 +261,7 @@ export default class XMPP extends Listenable {
                                         }
                                     });
                                 })
-                                .catch(logger.warn('Error getting features from lobby.'));
+                                .catch(() => logger.warn('Error getting features from lobby.'));
```

Comment:
is the error logged anywhere up? can you do `error => logger.warn('Error getting features from lobby.', error && error.message))` ?

Reply:
will do